### PR TITLE
#22733 - removed fields designed for just a single bot

### PR DESCRIPTION
**Pull request by justinwilaby**

### CODE REVIEW

File: packages/app/client/src/ui/editor/botSettingsEditor/index.tsx [hunk 1]
```diff
@@ -203,13 +203,6 @@ class BotSettingsEditor extends React.Component<BotSettingsEditorProps, BotSetti
     return (
       <GenericDocument style={ CSS }>
         <Column>
-          <MediumHeader className="bot-settings-header">Bot Settings</MediumHeader>
-          <TextInputField label="Bot name" value={ this.state.bot.name } required={ true } onChange={ this.onChangeName } />
```
Comment (eanders-MS) [inline packages/app/client/src/ui/editor/botSettingsEditor/index.tsx]: I think we want to keep bot name.

File: packages/app/client/src/ui/editor/botSettingsEditor/index.tsx [hunk 2]
```diff
@@ -203,13 +203,6 @@ class BotSettingsEditor extends React.Component<BotSettingsEditorProps, BotSetti
...
-          <TextInputField label="Endpoint URL" value={ this.state.endpoint.endpoint } required={ true } onChange={ this.onChangeEndpoint } />
-          <Row className="multiple-input-row">
-            <TextInputField label="MSA App Id" value={ this.state.endpoint.appId } onChange={ this.onChangeAppId } />
-            <TextInputField label="MSA App Password" value={ this.state.endpoint.appPassword } onChange={ this.onChangeAppPw } type="password" />
-          </Row>
```
Comment (tonyanziano) [inline packages/app/client/src/ui/editor/botSettingsEditor/index.tsx]: Remove the `onChange` handlers for these removed fields?

Comment (justinwilaby) [reply packages/app/client/src/ui/editor/botSettingsEditor/index.tsx]: +1
I wasn't sure if we'd need these once we start to support multiple bots in the settings

File: packages/app/client/src/ui/editor/botSettingsEditor/index.tsx (hunk 1 above)
Comment (justinwilaby) [reply packages/app/client/src/ui/editor/botSettingsEditor/index.tsx]: Added back the bot name and also installed validation for the required field.

File: packages/app/client/src/ui/editor/botSettingsEditor/index.tsx (hunk 2 above)
Comment (tonyanziano) [reply packages/app/client/src/ui/editor/botSettingsEditor/index.tsx]: We can always dredge them out of the commit history. 😃

**Pull request merged**